src/Microsoft.ML.Data/DataLoadSave/CompositeDataLoader.cs

```diff
@@ -582,10 +586,10 @@ public RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Random rand
             return View.GetRowCursorSet(predicate, n, rand);
         }
 
-        public SlotCursor GetSlotCursor(int col)
+        SlotCursor ITransposeDataView.GetSlotCursor(int col)
```

Comment:
>SlotCursor ITransposeDataView.GetSlotCursor [](start = 8, length = 43)

Ah, I see you figured out the explicit interface implementations after all. :) Good work.

Reply:
Thanks to your and Eric's comments.

---
In reply to: [248778138](https://github.com/dotnet/machinelearning/pull/2111#discussion_r248778138) [](ancestors = 248778138)